components/webhook-mgt/org.wso2.carbon.identity.webhook.metadata/src/main/resources/caep-event-profile.json

```diff
@@ -0,0 +1,43 @@
+{
+    "channels": {
+        "Security_Channel": {
```

Comment:
let's use camel case

Reply:
we can improve this file, so this file it self will define the profie type (or name)
And we can read all the files in this path and load the profiles

Reply:
And let's not give a name for these channels, channels should be an array of objects

Reply:
And actually let's avoid caep for the moment. we can use to to finalize the file structure, but let's not add it